SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -333,6 +382,15 @@ void ASpatialFunctionalTest::FinishTest(EFunctionalTestResult TestResult, const
 {
 	if (HasAuthority())
 	{
+		/*
+		 * Currently the Editor process will always be a Client, therefore only multi-cast
+		 * the failure message if running with multiple processes.
+		*/
+		if (TestResult == EFunctionalTestResult::Failed && GIsEditor == false)
```

Comment:
Is there no better way of checking if we are multiprocess?

Reply:
We are now using a function for this which gets the ULevelEditorPlaySettings and checks for multiple processes there.